python-squid/src/main/java/org/sonar/python/semantic/SymbolTableBuilderVisitor.java

```diff
@@ -387,4 +480,10 @@ public ClassVariableAssignmentVisitor(AstNode classTree) {
     }
 
   }
+
+  private static class Module {
```

Comment:
I think we can create a constructor and make fields final, so it's clear that this object is not mutable